lib/rails_cursor_pagination/paginator.rb

```diff
@@ -422,9 +456,9 @@ def sorted_relation
     # @return [String (frozen)]
     def id_column
```

Comment:
We should probably rename this method to `#primary_key_column` and also change its description, right?